src/main/java/com/rackspace/ceres/app/services/MetadataService.java

```diff
@@ -359,4 +361,33 @@ public MetricNameAndMultiTags getMetricNameAndTags(String m) {
         );
 
   }
+
+
+  /**
+   * Returns Tags based upon tenantId and metricName/metricGroup
+   *
+   * @param tenantHeader the tenant header
+   * @param metricName   the metric name
+   * @param metricGroup  the metric group
+   * @return the tags
+   */
+  public Mono<GetTagsResponse> getTags(String tenantHeader, String metricName,
+      String metricGroup) {
+    HashMap<String, String> tags = new HashMap<>();
+    if (!StringUtils.isBlank(metricName)) {
```

Comment:
Did you test all 3 here?:
1. omit metricName
2. metricName = null
3. metricName = ""